scanpy/tests/notebooks/test_pbmc3k.py

```diff
@@ -132,23 +160,22 @@ def test_pbmc3k(image_comparer):
     if adata[adata.obs["leiden"] == "4", "CST3"].X.mean() < 1:
         (  # switch clusters
             adata.obs["leiden"][adata.obs["leiden"] == "4"],
-            adata.obs["leiden"][adata.obs["leiden"] == "5"],
-        ) = ("5", "4")
+            adata.obs["leiden"][adata.obs["leiden"] == "3"],
+        ) = ("3", "4")
```

Comment:
5 <-> 3
<img width="324" alt="Screenshot 2024-01-24 at 12 39 49" src="https://github.com/scverse/scanpy/assets/43999641/8683dcb6-5116-40dd-90d4-a35cd61cf291">
<img width="277" alt="Screenshot 2024-01-24 at 12 39 03" src="https://github.com/scverse/scanpy/assets/43999641/658de9a8-5e93-4099-80d8-9cf338043736">

Reply:
if you manually order them in the first place, why reorder here?

Reply:
I didn't want to touch whatever this is doing since it's based on mean expression.